src/cli/keepassxc-cli.1

```diff
@@ -91,6 +94,9 @@ Path of the key file for the second database.
 .IP "--no-password-from"
 Deactivate password key for the database to merge from.
 
+.IP --yubikey-from <slot>"
```

Comment:
mismatched quotes, still seems to come out right though

Reply:
nice catch, fixed!